### PR TITLE
Remove remaining copies of W3C 3-clause BSD License

### PR DESCRIPTION
**Pull request by foolip**
In https://github.com/web-platform-tests/rfcs/blob/master/rfcs/simplify_license.md
we resolved to simplify the license to the plain 3-Clause BSD License,
with the support of the W3C, and this was implemented in
https://github.com/web-platform-tests/wpt/pull/11191.

It was an oversight that these copies of the license were left.

### CODE REVIEW

File: tools/wptrunner/wptrunner/update/sync.py [hunk 1]
```diff
@@ -10,37 +10,6 @@
 
 here = os.path.abspath(os.path.split(__file__)[0])
 
-bsd_license = """W3C 3-clause BSD License
```
Comment (foolip) [inline tools/wptrunner/wptrunner/update/sync.py]: @jgraham is this part of Gecko sync? Will there be any adverse impact of not continuing to write this file?

**Comment by foolip** [comment]
@web-platform-tests/wpt-core-team I believe this is a straightforward followup to finish https://github.com/web-platform-tests/rfcs/pull/18, but it's important to get these things right. If anyone thinks this requires a new RFC, please let me know.

File: tools/wptrunner/wptrunner/update/sync.py (hunk 1 above)
Comment (jgraham) [reply tools/wptrunner/wptrunner/update/sync.py]: I don't think so. This dates from when there was a choice of licenses and we wanted to indicate that we were taking the BSD option. In practice gecko doesn't use this code anyway, but I think servo does, so @jdm may want to be aware that it's changing.

**Review by jgraham — APPROVED** [review]

**Comment by jgraham** [comment]
@foolip do you want to land this?

**Comment by foolip** [comment]
@jgraham yes, now merged! :)

**Pull request merged**